scan/spec/test_command_generator_spec.rb

```diff
@@ -74,6 +74,38 @@
       end.to raise_error "Project file not found at path '/notExistent'"
     end
 
+    describe ":toolchain" do
+      it "should fail if :xctestrun and :toolchain is set" do
+        expect do
+          Fastlane::FastFile.new.parse("lane :test do
+            scan(
+              project: './scan/examples/standard/app.xcodeproj',
+              xctestrun: './path/1.xctestrun',
+              toolchain: 'com.apple.dt.toolchain.Swift_2_3'
+            )
+          end").runner.execute(:test)
+        end.to raise_error("Unresolved conflict between options: 'toolchain' and 'xctestrun'")
+      end
+      it "Supports toolchain" do
```

Comment:
This should be a part of the `:toolchain` `describe`, and should maybe be `passes the toolchain option to xcodebuild`